assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -38,23 +38,45 @@ import AdBlockerWarning from '../common/AdBlockerWarning';
 
 const { useSelect } = Data;
 
+/**
+ * Calculates the absolute change between two values.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} previous The previous value.
+ * @param {number} current  The current value.
+ * @return {number} The absolute change between values.
+ */
+const calculateAbsoluteChange = ( previous, current ) => {
+	const change = Number( current ) - Number( previous );
+	return change.toFixed( 2 );
```

Comment:
We shouldn't need to use `toFixed` in most cases since this is essentially number formatting which should be done by `numFmt` instead.

Without this, the function here essentially becomes `return current - previous` which seems unnecessary to make into a function 😄